### PR TITLE
Windows no longer officially unsupported - FAQ

### PR DESCRIPTION
**Pull request by brunobowden**
- Doesn’t quite say it’s officially supported

### CODE REVIEW

**Comment by brunobowden** [comment]
@advayDev1, @gocursor

File: FAQ.md [hunk 1]
```diff
@@ -259,19 +259,18 @@ For more details:
...
+Complete development on iOS is supported only on Mac OS X. This matches the
 [J2ObjC Requirements](http://j2objc.org/#requirements).
-In practice, the j2objcCycleFinder and j2objcTranslate tasks may work, though it is
-strongly suggested to use this for experimentation only and do all proper development
-on OS X. The team will reject any bugs related to systems that don't meet the
-[minimum requirements](README.md#minimum-requirements). You can attempt just
-j2objcCycleFinder and j2objcTranslate by adding the following line to your local.properties
-file only on Windows machines.
+
+It is recommended that Windows do the limited translation only build to reduce the chances
```
Comment (advayDev1) [inline FAQ.md]: "Windows users..."

Comment (advayDev1) [inline FAQ.md]: "translation-only"

File: FAQ.md [hunk 2]
```diff
@@ -259,19 +259,18 @@ For more details:
...
+of breaking the build on OS X. This can be done by the Windows developers configuring their
```
Comment (advayDev1) [inline FAQ.md]: "of breaking the iOS/OS X build"

**Comment by advayDev1** [comment]
lgtm - minor comments

File: FAQ.md (hunk 1 above)
Comment (brunobowden) [inline FAQ.md]: fixed

File: FAQ.md (hunk 2 above)
Comment (brunobowden) [inline FAQ.md]: done

**Comment by gocursor** [comment]
I am happy, that "Windows is no longer officially unsupported"... :-)

**Pull request merged**